config/samples/operator_v1beta1_cryostat.yaml

```diff
@@ -12,3 +12,8 @@ spec:
       labels: {}
       annotations: {}
       spec: {}
+  reportOptions:
+    replicas: 1
+    requests:
+      cpu: 1000m
+      memory: 512Mi
```

Comment:
Could you update the CSV with `make bundle`? Also if these values should be made defaults in the form view, could you add them to the CR in this annotation:
https://github.com/cryostatio/cryostat-operator/blob/0ee99b4242c5b53d2186ce56ec5af6531179b36f/config/manifests/bases/cryostat-operator.clusterserviceversion.yaml#L56-L67

Reply:
Yea, I'm not sure what the defaults should be - that really depends on the size of the recordings a user is generally handling. Maybe in the interest of keeping the initial deployed footprint small the default should be 0 replicas and no resources config.

Reply:
Makes sense